coin-pickup-game/src/index.js

```diff
@@ -23,12 +23,12 @@ const Themes = [
     }
 ]
 
-module.exports = class CoinPickupGame extends BasePlugin {
+export default class CoinPickupGame extends BasePlugin {
 
     /** Plugin info */
-    static get id()             { return 'vatominc-coin-pickup-game' }
-    static get name()           { return 'Coin Pickup Game' }
-    static get description()    { return 'A game where users can walk over coins to collect them.' }
+    static id = "vatominc-coin-pickup-game"
+    static name = "Coin Pickup Game"
+    static description = "A game where users can walk over coins to collect them."
 
     /** Current number of coins collected */
     score = 0
```

Comment:
The spawned coins are just objects and they don't have the coin component attached. This functionaly is not documented. The relevant code seems to be:
```javascript
            ['component:' + this.plugin.id + ':coin:' + 'pickup-sound']: this.getField('pickup-sound') || this.paths.absolute('collect.wav'),
            ['component:' + this.plugin.id + ':coin:' + 'score']: this.getField('score') || '',
            components: [
                { id: this.plugin.id + ':coin' }
            ]
```

Reply:
Yes you are right, we need to document that structure

Reply:
I would be happy to document it; the only thing is that it does not work. The spawned coins are just objects and they don't have the coin component attached. Probably something changed in the API?